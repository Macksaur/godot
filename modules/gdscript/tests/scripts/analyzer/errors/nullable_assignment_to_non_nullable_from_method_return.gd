func _get_result() -> Array[String]?:
    var arr: Array[String]? = null
    return arr

func test():
    var dict: Array[String] = _get_result()
