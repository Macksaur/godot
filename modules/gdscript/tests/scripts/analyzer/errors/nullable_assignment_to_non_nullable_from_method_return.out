GDTEST_ANALYZER_ERROR
Potentially unsafe to assign the nullable return of "_get_result()" to non-nullable "dict".
